commercial/app/views/contentapi/items.scala.html

```diff
@@ -44,10 +44,12 @@ <h3 class="ad-slot--paid-for-badge__header">@optSponsorLabel:</h3>
                                             @fragments.items.elements.facia_cards.image(trailPictureContainer, true)
                                         }
                                         <div class="fc-item__content">
-                                            <h2 class="fc-item__title">
-                                                @insertItemMediaIcon(item)
-                                                @item.webTitle
-                                            </h2>
+                                            <div class="fc-item__header">
```

Comment:
Is it necessary to add another div? Can't you include that font inside fc-item__title with the same result?